src/bp/core/config/config-loader.ts

```diff
@@ -61,6 +64,22 @@ export class ConfigProvider {
       config.pro.licenseKey = process.env.BP_LICENSE_KEY || config.pro.licenseKey
     }
 
+     [
+      ['BP_PORT', 'HTTTPSERVER.PORT'],
+      ['PORT', 'HTTTPSERVER.PORT'],
```

Comment:
`PORT` is provided automatically by some hosts (ex: heroku), so that warning will always be present